### PR TITLE
rename to controller-cursor, bump for 0.5.0, move docs -> examples (fixes #3)

### LINKED ISSUE #3: rename the component a more generic way
This is just a suggestion, but since the component works also with other types of controllers (I tested it with Oculus Touch) it would make more sense to rename it to something more generic.

### PR DESCRIPTION
**Pull request by ngokevin**
Tested with hand-controls, works well for that. Don't need to be specific to Vive. Haven't tested with Touch yet, but just matter of possibly positioning the laser and making it configurable.

Will need help renaming the repo + publishing if it sounds good, then we can add to Registry.

### CODE REVIEW

**Comment by ngokevin** [comment]
I think I started from cursor.js as a base, so the more recent fixes should be there?

**Comment by bryik** [comment]
Looks great!

The code in the repo is basically a copy of whatever cursor.js looked like July of last year, so if you started from a newer version the fixes I was talking about in [issue #1](https://github.com/bryik/aframe-vive-cursor-component/issues/1) should be covered.

Are there any plans to support a controller cursor in the core? As @noerw pointed out [recently](https://github.com/bryik/aframe-vive-cursor-component/issues/5) the code duplication is unfortunate.

File: package.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
-  "name": "aframe-vive-cursor-component",
-  "version": "0.1.2",
-  "description": "Vive Cursor component for A-Frame.",
+  "name": "aframe-controller-cursor-component",
+  "version": "0.3.0",
```
Comment (bryik) [inline package.json]: Why version 0.3.0?

**Comment by ngokevin** [comment]
I've fixed the version.

Perhaps one day we can make generalizable ray input / cursor component and have it more core, but it's a large chunk of work.

**Comment by bryik** [comment]
>Perhaps one day we can make generalizable ray input / cursor component and have it more core, but it's a large chunk of work.

OK.

**Comment by bryik** [comment]
All right, I've changed the repo's name and published to NPM.

Now to figure out how to deprecate the old NPM package...

**Comment by ngokevin** [comment]
Push a version with a modified README and console.log?

**Pull request merged**